.opencode/commands/implement-oxlint-rules.md

```
@@ -0,0 +1,132 @@
+
+For example:
+- `eslint/eqeqeq` → grep for `"eqeqeq"` in `eslint-global.flat-config.ts`
+- `typescript/dot-notation` → grep for `"@typescript-eslint/dot-notation"` in `eslint-typescript.flat-config.ts`
+
+If found, extract the full value (e.g. `"error"`, `"off"`, or `["error", { ... }]`).
+
+#### c. Present findings to user
+
+Display in this format:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)